<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>231</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>231</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Step by Step</string>
    </property>
    <item>
     <property name="text">
      <string>Step by Step</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DDA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bresenham</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bresenham Circle</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>750</width>
      <height>750</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Graphic</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>211</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Point 1</string>
       </property>
       <widget class="QSpinBox" name="y1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-15</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="x1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-15</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
       <widget class="QLabel" name="x1_label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="X_2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Point 2</string>
       </property>
       <widget class="QSpinBox" name="x2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-15</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="y2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-15</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
       <widget class="QLabel" name="x2_label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="y2_label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Radius</string>
       </property>
       <widget class="QSpinBox" name="radius">
        <property name="geometry">
         <rect>
          <x>-2</x>
          <y>40</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="zoomIn">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zoom in</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-PKG_lab4-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/zoom-in.png</normaloff>../build-PKG_lab4-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/zoom-in.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zoom out</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-PKG_lab4-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/zoom-out.png</normaloff>../build-PKG_lab4-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/zoom-out.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Basic Raster 
Algorithms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Time (ns): </string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>650</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-PKG_lab4-Desktop_Qt_5_15_2_MinGW_64_bit-Debug/checkmark.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
